src/main/resources/modules/female_reproduction.json

```diff
@@ -147,7 +137,6 @@
         "indicate the number out of 100 women who experience planned or unplanned ",
         "pregnancy within the first year of use. These percentages are scaled for ",
         "a monthly time step: ",
-
```

Comment:
I'm conflicted on formatting within remarks.  Should we retain these paragraph markers (perhaps with a line with `"",`)?  We'll just lose this information if we merge this in now.

Reply:
I'm ok with losing these blank lines. As long as it doesn't reformat the content within quotes, ex keeps the table formatting shown here, it's fine.